src/api/v2/agents/handlers/join.ts

```
@@ -13,7 +13,49 @@ function buildInviteUrl(slug: string): string {
+ *   -H "Content-Type: application/json" \
+ *   -H "X-Force-Error: 502" \
+ *   -d '{"slug": "test-slug"}'
+ * ```
+ */
 export async function joinHandler(req: Request, res: Response) {
+  // Force error responses for testing — see JSDoc above for usage
+  const forceError = req.headers["x-force-error"];
+  if (forceError === "502" || forceError === "503" || forceError === "504") {
+    const status = Number(forceError);
+    const errorMap: Record<number, { error: string; message: string }> = {
+      502: { error: "AGENT_PROVISION_FAILED", message: "Failed to provision agent" },
```

ESLint

[error] Replace `·error:·"AGENT_PROVISION_FAILED",·message:·"Failed·to·provision·agent"` with `⏎········error:·"AGENT_PROVISION_FAILED",⏎········message:·"Failed·to·provision·agent",⏎·····`

(prettier/prettier)